### PR TITLE
Redesigning the array HTML repr for clarity

### LINKED ISSUE #9447: Redesigning the array HTML repr for clarity
While the current HTML repr provides very useful information, it can also be _confusing_ for new users. This is due to the fact we currently use a table to represent information (`Type`, `Count`) that is not directly comparable for `Array` and `Chunk` columns

Screen Shot 2022-08-31 at 11 04 07 AM

To improve clarity, it might be useful to get the stuff that’s not directly comparable out of the table. The new HTML repr would 

- keep `Bytes` and `Shape` in a table:

   Screen Shot 2022-08-31 at 11 23 21 AM

   
    This describes the same thing (bytes and shape) across both arrays and chunks.

- and then, outside of the table we could have something along these lines 

  ```
  Dask graph: X chunks in Y tasks
  Data type: float32 numpy.ndarray
  ```

@dask/array, how do folks feel about this?

_The suggestion above came from an offline conversation with @freeman-lab._

### PR DESCRIPTION
**Pull request by ognis1205**
Remove `Count` and `Type` rows from the `dask.Array._repr_html_`.

Signed-off-by: Shingo OKAWA <shingo.okawa.g.h.c@gmail.com>

- [x] Closes #9447 
- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

Admins can comment `ok to test` to allow this one PR to run or `add to allowlist` to allow all future PRs from the same author to run.

File: dask/widgets/templates/array.html.j2
Comment (GenevieveBuckley) [inline dask/widgets/templates/array.html.j2:3]: "Chunks" should probably not be capitalized here since it is in the middle of a sentence.

**Review by GenevieveBuckley — CHANGES_REQUESTED** [review]
Thank you for this PR @ognis1205!

I've realized that moving information into the caption makes it a lot less eye-catching. Can I suggest for important information like the type, we instead keep it in the table, but let it span across both columns?

The HTML `colspan` allows control of this, eg:
```html
<td colspan="2"> {{ array.dtype }} {{ array._meta | type | typename }} </td>
```

**Comment by GenevieveBuckley** [comment]
Screenshots are probably helpful for any reviewers who don't have time to check out the code. Here is a screenshot of the output from the original PR

Screen Shot 2022-09-29 at 11 24 43 am

**Comment by ognis1205** [comment]
Thank you for the review @GenevieveBuckley!
I will fix the issues and let you know when I complete. Besides that, I will upload the screenshot of the outputs of the PR as well.

File: dask/widgets/templates/array.html.j2
Comment (ognis1205) [reply dask/widgets/templates/array.html.j2:3]: In that case, should I change "Graph Layer" into "graph layer" in this line as well?

https://github.com/dask/dask/blob/main/dask/array/core.py#L1648

Screen Shot 2022-09-30 at 14 03 34

**Comment by ognis1205** [comment]
@GenevieveBuckley 
I fixed the issues. Some checks are failed but I think they are not relevant to this request:

https://github.com/dask/dask/actions/runs/3165861097/jobs/5155195568#step:7:22379
https://github.com/dask/dask/actions/runs/3165861097/jobs/5155195603#step:7:22320

Could you please review the PR again? Here are screenshots of the resulting HTML:

Screen Shot 2022-10-02 at 4 19 46

Screen Shot 2022-10-02 at 4 20 03

Thank you.

File: dask/widgets/templates/array.html.j2
Comment (GenevieveBuckley) [reply dask/widgets/templates/array.html.j2]: I think that's a good idea, yes

**Review by GenevieveBuckley — APPROVED** [review]
This looks good to me. 

I like the way the screenshots look now, to me it seems a bit clearer. Perhaps @andersy005  would like to weigh in? 

I agree, the test failures are unrelated to this PR, and should not prevent merging. I don't have write access to merge, but I'll mark this approved, and another Dask maintainer should come by soon.

**Comment by ognis1205** [comment]
Thanks for your kind cooperation @GenevieveBuckley !

**Comment by andersy005** [comment]
Thank you for working on this l, @ognis1205! I agree with @GenevieveBuckley about keeping the dtype and graph information closer to the rest of items in the table. This looks solid to me but I'll let others chime in.

**Comment by ncclementi** [comment]
add to allowlist

**Review by phobson — COMMENTED** [review]
Thanks!

**Review by phobson — APPROVED** [review]
Thanks!

**Comment by ncclementi** [comment]
@ognis1205 Thank you very much for your contribution and thank you for your patience. @GenevieveBuckley and @andersy005 Thank you for your reviews.

This one goes in. CI issue is unrelated

**Pull request merged**
